sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingSymmetricHashJoinExec.scala

```diff
@@ -382,6 +398,7 @@ case class StreamingSymmetricHashJoinExec(
             leftSideJoiner.removeOldState() ++ rightSideJoiner.removeOldState()
           case LeftOuter => rightSideJoiner.removeOldState()
           case RightOuter => leftSideJoiner.removeOldState()
+          case FullOuter => Iterator.empty
```

Comment:
Let's also add comments for FullOuter in: https://github.com/apache/spark/pull/30395/files#diff-6cd66da710d8d54025c1edf658bbec5230e8b4e748f9f2f884a60b1ba1efed42R395

Reply:
@xuanyuanking - agree, updated.